src/es/modules/callout.js

```diff
@@ -159,6 +172,15 @@ export class TourCallout extends Callout {
     }
     return Object.assign(opts, tourOpts);
   }
+
+ /**
+  * Renders markup of the callout and inserts callout element into the DOM
+  * @override 
+  */
+  render() {
+    super.render();
+    Utils.addClass(this.el, 'tour-' + this._tour.id);
```

Comment:
I can't comment on line 145, but now that this logic is here, line 145 should be removed.